src/System.IO.Abstractions.TestingHelpers/MockFileInfo.cs

```diff
@@ -178,6 +178,14 @@ public override DateTime LastWriteTimeUtc
             }
         }
 
+#if NET6_0_OR_GREATER
+        /// <inheritdoc />
+        public override string LinkTarget
+        {
+            get { return null; }
```

Comment:
I need guidance on this. Where would we set/get this data? I'll take a look tonight to see, but if someone can point me in the right direction is might save some time.

Reply:
Same comments as above.

Reply:
Done